Makefile

```diff
@@ -5,7 +5,6 @@ MANPATH=/usr/share/man
 dirs = lib man
```

Comment:
Don't see the point of maintaining MAKEFILEs given they duplicate setuptools functionality but I digress.

Reply:
I like the "make tar" part with the version consistency check between the spec file and the source. But yes we could drop install.